modules/util.py

```diff
@@ -44,13 +159,28 @@ class BrowserActions:
     def __init__(self, driver: Firefox):
         self.driver = driver
 
+    def clear_and_fill_no_additional_keystroke(self, webelement: WebElement, term: str):
```

Comment:
You could modify `clear_and_fill` with a `press_enter` attr with `True` as the default, but this works too. Again, whichever you want is ok.

Reply:
i am going to leave this one because there are other tests that use the original method